src/Components/RoutesWrapper.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate, Routes, Route } from 'react-router-dom';
+import '../Style/myReservation.css';
+import { getExperiencesData } from '../Redux/Slices/ExperiencesSlice';
+import { getUserReservations } from '../Redux/Slices/reservationsSlice';
+import MainPage from './MainPage';
+import ReservationForm from './ReservationForm';
+import MyReservations from './MyReservations';
+import ExperienceCard from './ExperienceCard';
+
+function RoutesWrapper() {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const status = useSelector((state) => state.experiences.status);
+  const userStatus = useSelector((state) => state.users.status);
+  const { id } = useSelector((state) => state.users.user);
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(getExperiencesData());
+    }
+  }, [dispatch, status]);
+
+  useEffect(() => {
+    if (status === 'succeeded') {
+      dispatch(getUserReservations(id));
+    }
+  });
+
+  if (userStatus === 'Authenticated') {
+    navigate('/login');
+  }
+
+  return (
+    <Routes>
+      <Route path="MainPage" element={<MainPage />} />
```

Comment:
Kindly fix this route, and add a `/` before `MainPage` like in the other routes.